packages/gatsby-cli/package.json

```diff
@@ -40,7 +40,7 @@
     "prompts": "^2.3.2",
     "react": "^16.8.0",
     "redux": "^4.0.5",
-    "resolve-cwd": "^2.0.0",
+    "resolve-cwd": "^3.0.0",
```

Comment:
Just so there is paper trail (not something to act on):
Breaking changes in 2->3 ( from https://github.com/sindresorhus/resolve-cwd/releases/tag/v3.0.0 )
```
Breaking:

- Require Node.js 8
- Return undefined instead of null when the module can't be found when using resolveCwd.silent()
```

Both are good and don't require any changes from us - we already require node 10 and `resolveCwd.silent()` change doesn't impact us because we just check for falsiness and not concrete values (and both `undefined` and `null` are falsy)